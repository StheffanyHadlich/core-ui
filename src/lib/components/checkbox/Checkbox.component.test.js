import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/checkbox/__snapshots__"
});
