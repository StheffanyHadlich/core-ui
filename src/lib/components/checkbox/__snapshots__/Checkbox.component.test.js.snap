// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Checkbox Default 1`] = `
<div
  className="sc-htpNat iharJH"
>
  <h3
    className="sc-bxivhb iDJTyr"
  >
    Checkbox checked
  </h3>
  <label
    checked={true}
    className="sc-checkbox sc-bwzfXH buIIzS"
  >
    <input
      checked={true}
      data-cy="checked_checkbox"
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="fas fa-check"
    />
    <span
      className="text sc-bdVaJa hERZIf"
    >
      interested?
    </span>
  </label>
  <h3
    className="sc-bxivhb iDJTyr"
  >
    Checkbox unchecked
  </h3>
  <label
    checked={false}
    className="sc-checkbox sc-bwzfXH fiqgwR"
  >
    <input
      checked={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="fas fa-check"
    />
    <span
      className="text sc-bdVaJa hERZIf"
    >
      interested?
    </span>
  </label>
  <h3
    className="sc-bxivhb iDJTyr"
  >
    Checkbox disabled
  </h3>
  <label
    checked={false}
    className="sc-checkbox sc-bwzfXH dIukKP"
    disabled={true}
  >
    <input
      checked={false}
      disabled={true}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="fas fa-check"
    />
    <span
      className="text sc-bdVaJa hERZIf"
    >
      interested?
    </span>
  </label>
</div>
`;
